docs/dificil/small_to_large.md

```diff
@@ -1,7 +1,233 @@
----
-title: ""
----
 
-# Alte probleme și resurse utile
 
-* [Small-to-large](https://usaco.guide/plat/merging?lang=cpp)
\ No newline at end of file
+# Small to large
+
+**Autor** : Radu Mocănașu
+
+## Cunoștințe necesare:
```

Comment:
In demonstratie vorbesti de principiul de la paduri de multimi disjuncte, ceea ce e corect, deci ar trebui sa fie DSU la prerequisites (un alt nume pentru small to large e dsu on trees)